Dockerfile

```diff
@@ -11,19 +11,20 @@ COPY ui /ui
 RUN npm run build
 
 FROM alpine:3.15
-LABEL org.opencontainers.image.title="alfresco-extension" \
+LABEL org.opencontainers.image.title="Alfresco" \
```

Comment:
I would probably add the information on the image that is a Community version. Alfresco CE? Alfresco Community?

Reply:
Alfresco Community